npc/re/merchants/pet_trader.txt

```diff
@@ -0,0 +1,177 @@
+//===== rAthena Script =======================================
+//= Pet Trader NPC
+//===== Description: =========================================
+//= [Official Conversion]
+//= Exchanges Cute Pet Tickets for Pet Eggs.
+//===== Additional Comments: =================================
+//= 1.0 First version [Latiosu]
+//============================================================
+
+prontera,210,209,5	script	Pet Trader#1	4_M_JPNOJI,{
+
+	disable_items;
+	mes "[Pet Trader]";
+	mes "Making a living through business ain't easy.";
+	mes "I get real bad motion sickness, yet today I got caught up in a storm on my way here.";
+	next;
+	mes "[Pet Trader]";
+	mes "It was such a horrible experience. Thanks to that, I don't feel so good. Ughh...";
+	next;
+	mes "[Pet Trader]";
+	mes "Anyway, did you come here looking for a Cute Pet?";
+	mes "You've brought a ticket with you, right?";
+	next;
+	switch (select("Examine your tickets:Cancel")) {
+	case 1:
+		.@menu$ = "";
+		.@ticketIndices = 0;
+		.@count = 0;
```

Comment:
```suggestion
```

Reply:
I'm not sure of the intention behind removal here. Are you able to clarify @Atemo ?

Reply:
The default value of unset variables are 0 for integer and "" for string, no need to declare them in rathena npc script

Reply:
I had another look at the [docs](https://github.com/rathena/rathena/blob/master/doc/script_commands.txt#L474-L479) and realised what my misunderstanding was.
I misinterpreted scope variables to be 'block scope' (like in Java or C) rather than 'script scope'.